setup.py

```diff
@@ -41,7 +41,8 @@ def mecab_config(arg):
 setup(name = "mecab-python3",
     version = '0.8.3',
     description = 'python wrapper for mecab: Morphological Analysis engine',
-    long_description = read_file('README.rst'),
+    long_description = read_file('README.md'),
+    long_description_content_type = ”text/markdown”,
```

Comment:
You have curly quotes here, which would cause all invocations of setup.py to fail.  It's a small thing, except that it means you didn't test this patch at all.  A change like this needs to be tested with at least `setup.py build` and `setup.py sdist` + manual verification of the contents of the generated tarball.

Reply:
oops, it's my mistake. will fix and test.